advanced/load_save.rst

```diff
@@ -0,0 +1,247 @@
+message being displayed ("Save file <id> doesn't exist").
+
+Files are loaded from the folder specified in the Godot's Project Settings
+under ``escoria/main/game_version``. If not changed, the default is '<game
+folder>/saves'.
+
+.. code-block:: gdscript
+
+  var slot: int = 1
+  var savename: String: "Blocked in room before puzzle"
+
+  # This will save the game in the file /saves/save_001.tres
+  escoria.save_manager.save_game(slot, savename)
+
+Loading and saving settings
+---------------------------
+
+Game settings are loaded when the game is run. If no settings file exists, a
+new one is created with default values.
+
+It is not required to manage settings saving and loading if the player is not
+supposed to change one of them. Otherwise, settings should be saved. It is the
+duty of the game developer to manage this in a settings UI included in the game
+menus.
+
+Settings files are stored in a :doc:`ESCSaveSettings</api/ESCSaveSettings>`
+resource. resource file.
+
+The default data included in the settings file are:
+
+- ``text_lang``: the language of the text displayed in the game.
+
+- ``voice_lang``: the speech language.
+
+- ``speech_enabled``: a boolean value indicating whether audio speech is
+  enabled in the game.
+
+- ``master_volume``: the global volume value (a value between 0 and 100).
+
+- ``music_volume``: the volume of the music (a value between 0 and 100).
+
+- ``sfx_volume``: the  sound effects volume, ie. all sounds that are not music
+  or speech (a value between 0 and 100).
+
+- ``speech_volume``: the speech volume (a value between 0 and 100)
+
+- ``fullscreen``: a boolean value indicating whether the game is fullscreen or
+  not.
+
+- ``skip_dialog``: a boolean value indicating whether dialogue skipping is
+  allowed or not.
+
+- ``custom_settings``: a dictionary containing any custom data to be saved. For
+  example, the screen resolution could be added in this field. See `Saving
+  custom data in save game and settings files`_.
+
+Saving custom data in save game and settings files
+--------------------------------------------------
+
+As indicated above, both save game and settings files provide a specific field
+allowing the game developer to add custom data to these files. This section
+describes how to use these fields.
+
+Adding custom data to save game files
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The :doc:`ESCSaveGame </api/ESCSaveSettings>` data structure provides a
+dictionary variable ``custom_data`` which custom data can be added to.
+
+This feature requires a ``get_custom_data()`` function to be implemented in the
+``game.gd`` script of the game (extending ``ESCGame``). This function must
+return a dictionary value that contains the custom data to be saved.
+
+When the ``ESCSaveManager.save_game()`` function is called, the
+``ESCGame.get_custom_data()`` function is automatically called and the returned
+dictionary is then saved in the ``custom_data`` field of the save game file.
+
+
```

Comment:
Please remove the extra newline here.

Reply:
Submitted a suggested change for @StraToN to approve that fixes this.